conda-store-server/conda_store_server/worker/app.py

```diff
@@ -71,9 +71,18 @@ def start(self):
         argv = [
             "worker",
             "--loglevel=INFO",
-            "--beat",
         ]
 
+        # The default Celery pool requires this on Windows. See
+        # https://stackoverflow.com/questions/37255548/how-to-run-celery-on-windows
+        if sys.platform == "win32":
+            os.environ.setdefault('FORKED_BY_MULTIPROCESSING', '1')
```

Comment:
It makes me worried that we rely on a library that has no official Windows support.

It's worth discussing the level of Windows support we expect to provide ourselves and possibly consider alternatives to celery.

I did attempt to test this by running my concurrency test from https://github.com/conda-incubator/conda-store/pull/567/commits/3fc0e1425fae17f1bae8a1f35421e9c77f553059. I also parameterized it by all pools from `get_available_pool_names`. All of them failed. This could be for a number of reasons, but it's not worth investigating at the moment due to 50% of the testsuite failing for me. Also: pytest celery fixture might require additional tweaks to get working.

Before we start talking about celery or concurrency, I suggest we get the rest of the testsuite working on Windows.

Reply:
Most of the other pools fail because they aren't actually concurrent on Windows, and one of the tasks blocks all the others (I think the watch task, but I'm not completely sure). But even if that weren't the case, conda-store would be very slow without concurrent tasks due to some slow tasks like updating channels.